zuul.d/jobs.yaml

```diff
@@ -760,6 +760,27 @@
       ansible_test_command: network-integration
       ansible_test_integration_targets: "openvswitch_.*"
 
+- job:
+    name: ansible-test-network-sanity-openvswitch
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-network-openvswitch-appliance
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+      - playbooks/ansible-test-sanity-base/pre.yaml
```

Comment:
we shouldn't need any playbooks/ansible-test-sanity-base files, we can just add flags below in the var section. and just use playbooks from ansible-test-base